tox.ini

```diff
@@ -27,7 +27,7 @@ deps =
     flake8
     isort
 commands =
-    black --check django_prometheus/
```

Comment:
nope.
CI should just check if the code is properly formatted. not do the formatting.

Reply:
no
it will never pass test because https://pypi.org/project/black/

```
--check                         Don't write the files back, just return the
                                  status.  Return code 0 means nothing would
                                  change.  Return code 1 means some files
                                  would be reformatted.  Return code 123 means
                                  there was an internal error.
```
it always returns 1 if something was changed

```
would reformat /home/travis/build/korfuri/django-prometheus/django_prometheus/conf/__init__.py
would reformat /home/travis/build/korfuri/django-prometheus/django_prometheus/cache/metrics.py
would reformat /home/travis/build/korfuri/django-prometheus/django_prometheus/db/metrics.py
would reformat /home/travis/build/korfuri/django-prometheus/django_prometheus/migrations.py
would reformat /home/travis/build/korfuri/django-prometheus/django_prometheus/models.py
would reformat /home/travis/build/korfuri/django-prometheus/django_prometheus/middleware.py
Oh no! 💥 💔 💥
6 files would be reformatted, 40 files would be left unchanged.
ERROR: InvocationError for command /home/travis/build/korfuri/django-prometheus/.tox/py37-lint/bin/black --check django_prometheus/ (exited with code 1)
```